bbb-learning-dashboard/src/components/QuizzesTable.jsx

```
@@ -318,40 +379,29 @@ const QuizzesTable = (props) => {
-            id: 'app.learningDashboard.quizzes.noResponse',
-            defaultMessage: 'No response',
-          });
-        }
-        return isCorrect ? intl.formatMessage({
-          id: 'app.learningDashboard.quizzes.correct',
-          defaultMessage: 'Correct',
-        }) : intl.formatMessage({
-          id: 'app.learningDashboard.quizzes.incorrect',
-          defaultMessage: 'Incorrect',
-        });
+        const [, userAnswers] = params?.value;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)